gmso/utils/schema/ff-gmso.xsd

```diff
@@ -250,22 +249,6 @@
             <xsd:selector xpath="AtomTypes/AtomType"/>
             <xsd:field xpath="@name"/>
         </xsd:unique>
-        <xsd:unique name="bondtype_names">
-            <xsd:selector xpath="BondTypes/BondType"/>
-            <xsd:field xpath="@name"/>
-        </xsd:unique>
-        <xsd:unique name="angletype_names">
-            <xsd:selector xpath="AngleTypes/AngleType"/>
-            <xsd:field xpath="@name"/>
-        </xsd:unique>
-        <xsd:unique name="dihedraltypes_names">
-            <xsd:selector xpath="DihedralTypes/DihedralType"/>
-            <xsd:field xpath="@name"/>
-        </xsd:unique>
-        <xsd:unique name="pairpotentialtypes_names">
-            <xsd:selector xpath="PairPotentialTypes/PairPotentialType"/>
-            <xsd:field xpath="@name"/>
-        </xsd:unique>
```

Comment:
Why is this section removed?

Reply:
This section seems to be checking that there are no duplicate names for these potentials. But I don't thing we should be checking the names at all, because then we have like 50 dihedrals that have to be named PeriodicTorsionType-1...50, and if any of them are not labeled this way than it'll fail to load. However, the real key to checking for unique tyeps is checking that no combinations of the types and classes that make up the components of the connection type. I'm not certain why these were implemented this way at all, because in Foyer XML you don't need unique names for each type.

Reply:
Yeah, I think I am ok with that. Let's comment that section out before we can find a way to validate the uniqueness of these types via atomtype/atomclass